source/Halibut.sln

```diff
@@ -1,6 +1,7 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26228.4
+﻿
```

Comment:
Every time I open Halibut in visual studio it changes the solution file 🙄 I can revert if needed